project/SensibleSettings.scala

```diff
@@ -13,7 +13,8 @@ object Sensible {
 
   lazy val settings = Seq(
     ivyLoggingLevel := UpdateLogging.Quiet,
-    conflictManager := ConflictManager.strict,
+    // FIXME: how to be less strict for typesafe config? slick and akka pull in different versions
+    //conflictManager := ConflictManager.strict,
```

Comment:
No, they probably are compatible but `update` fails with this when the line is active:

```
[info] Updating {file:/home/johannes/git/opensource/ensime/}core...
[error] com.typesafe#config;1.2.1 (needed by [com.typesafe.slick#slick_2.11;3.1.1]) conflicts with com.typesafe#config;1.3.0 (needed by [com.typesafe.akka#akka-actor_2.11;2.4.4])
[trace] Stack trace suppressed: run last core/*:update for the full output.
[error] (core/*:update) org.apache.ivy.plugins.conflict.StrictConflictException: com.typesafe#config;1.2.1 (needed by [com.typesafe.slick#slick_2.11;3.1.1]) conflicts with com.typesafe#config;1.3.0 (needed by [com.typesafe.akka#akka-actor_2.11;2.4.4])
[error] Total time: 0 s, completed Apr 15, 2016 12:40:50 PM
```